File: js/matrices.js
let nombre = prompt("Ingrese el nombre:");
let apellido = prompt("Ingrese el apellido:");
let dni = prompt("Ingrese su DNI:");

let telefonos = [];

while (true) {
    let telefono = prompt("Ingrese su teléfono:");
    telefonos.push(telefono);

    let agregarNumero = prompt("¿Quiere agregar otro número? 1.Sí, 2.No:");

    if (agregarNumero === "2") {
        break;
    }
}

let persona = [nombre, apellido, dni, telefonos];

console.log(persona);

File: python/matrices.py
nombre = input("Ingrese el nombre: ")
apellido = input("Ingrese el apellido: ")
dni = input("Ingrese su dni: ")

telefonos = []


while True: 
    
    telefono = input("Ingrese su telefono: ")
    
    telefonos.append(telefono)
    
    agregarNumero = input("Quiere agregar otro numero | 1.Si, 2.No: ")
    
    if agregarNumero == "2":
        break 

persona = [nombre, apellido, dni, telefonos]

print(persona)  